resources/js/views/NoCookies.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div v-if="stateNotFound">
+    <p>Kunne ikke autentisere. Vennligst prøv igjen.</p>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "NoCookiesView",
+  props: {
+    state: "",
+    nonce: "",
+    targeturl: "",
+    storagetarget: ""
+    
+  },
+  data() {
+    return {
+      stateNotFound : false
+    }
+  },
+  created() {
+    document.body.style.backgroundColor = "#eaeaea"
+    let platformOIDCLoginURL = "https://bibsys.test.instructure.com/api/lti/authorize_redirect?"
+    let platformOrigin = new URL(platformOIDCLoginURL).origin;
+    let frameName = this.storagetarget;
+    let parent = window.parent || window.opener;
+    let targetFrame = frameName === "_parent" ? parent : parent.frames[frameName];
+    const redirect = () => {
+      window.location.href = platformOIDCLoginURL + this.targeturl
+    }
+    
+    //this is not recognized inside eventlistener
+    var self = this
+    window.addEventListener('message', function (event) {
+      // This isn't a message we're expecting
+      
+      if (typeof event.data !== "object"){
+        console.log("Invalid datatype")
+        this.stateNotFound = true;
+        return;
+      }
+      
+      // Validate it's the response type you expect
+      if (event.data.subject !== "lti.put_data.response" && event.data.subject !== "lti.get_data.response") {
+        console.log("Invalid subject")
```

Comment:
Her er det flere console.log() statements jeg antar er for debug. Disse kan vi få fjernet til produkjson.

Reply:
Tanken var at det er fint å kunne vite hva den feiler på om den mottar en ugyldig melding. Kan legge det inn i en string som printes i viewet om det feiler.